docs/deploy.md

```diff
@@ -2,54 +2,53 @@
 [:arrow_left: Back to USKPA
 Documentation](../docs)
 
-Two instances of the USKPA website are hosted by [Heroku].
+All instances are hosted by [Heroku].
 
-* Staging - https://uskpa-staging.herokuapp.com/
-* Production - https://uskpa.herokuapp.com/
+* Development - https://uskpa-dev.herokuapp.com/
+* Production - TBD
 
 ### Heroku
 
-At this early stage we're using the [Heroku CLI]
-to manage all user-facing instances of the application. In the future we
-will transition to automated deployments via continuous integration.
+We use the [Heroku CLI], Heroku dashboard, and Heroku Github
+Integration to manage instances and automate releases.
+
+Currently, the [master] branch is deployed to the `Development` environment
+on each commit for which the unit test suite executes without error.
 
 Please see the Heroku CLI docs on [deploying with git](https://devcenter.heroku.com/articles/git).
 
 **Note:** Interacting with the specific Heroku Applications
 dicussed below is limited to those with the necessary permissions.
 
-### Staging
-*Heroku app: uskpa-staging*
+### Development
+*Heroku app: uskpa-dev*
 
-Deployed from: [master](https://github.com/18F/uskpa/tree/master)
+Deployed from via Heroku Github Deploy: [master]
```

Comment:
Maybe a typo, should this be "Deployed via Heroku GitHub Deploy from: [master]"?